framework/auth/core.py

```diff
@@ -346,7 +348,7 @@ class User(GuidStoredObject, AddonModelMixin):
     # Social links
     social = fields.DictionaryField(validate=validate_social)
     # Format: {
-    #     'personal': <personal site>,
+    #     'profileWebsites': <list of profile websites>
```

Comment:
I believe this was discussed at the beginning of this PR, and personalWebsites was considered a bit confusing for internal use.  Personal to Whom?